cmake/KokkosConfig.cmake.in

```diff
@@ -1,9 +1,6 @@
 # Compute paths
 @PACKAGE_INIT@
 
-GET_FILENAME_COMPONENT(Kokkos_CMAKE_DIR "${CMAKE_CURRENT_LIST_FILE}" PATH)
```

Comment:
We need to keep this line because we use `Kokkos_CMAKE_DIR` to find the `KokkosTargets` file.

Reply:
Fixed.  Moved closer to the point it is used and unset after.